react/client/public/locales/en/translation.json

```diff
@@ -0,0 +1,13 @@
+{
+  "disclaimerText": {
+    "0": "Please make sure you are eligible for getting your record cleared. If you need help with this please email placeholder@website.com.",
+    "1": "Be aware your statement is stored locally on your device, and never transmitted to our servers or any third-party",
+    "2": "This software is developed by a non-governmental third party, Code for America.",
+    "3": "Please also review our Privacy Policy and Terms of Use before beginning.",
+    "4": "Please allocate at least 30 minutes of time to complete this.",
+    "5": "If at anytime you are confused please click the question mark button for guidance."
+  },
```

Comment:
i have a suspicion that we can still do it on one line and use a newline character `\n` and have it work (might need to style it with `white-space: pre-line;`). The only awkward thing is translators would need to recognize the newline but we won't concern ourselves too much about it right now.

Reply:
Beautiful, it works, thanks, that's a new one for me